src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -1090,4 +1097,73 @@ public function setEchoFormat($format)
     {
         $this->echoFormat = $format;
     }
+
+    /**
+     * Compile the switch statements into valid PHP.
+     * @param  string  $variable
+     *
+     * @return string
+     */
+    public function compileSwitch($variable)
+    {
+        return "<?php switch({$variable}){ ";
```

Comment:
without doubt, but if you close it ... even the tab after the @swtich will be printed as <?php echo '   '?> and it will cause problems. unless you write it that way
`@switch($var)
@case(10)
@breakcase
     @case(3)
      @breakcase`